src/pages/benefits/benefitFormUI/BenefitApplicationForm.tsx

```
@@ -116,14 +114,14 @@ const BenefitApplicationForm: React.FC = () => {
     const docSchemaStatic =
       documentTag?.list
         ?.filter((item: any) => item?.descriptor?.code === "mandatory-doc")
-        ?.map((item: any) => JSON.parse(item.value)) || [];
+        ?.map((item: any) => JSON.parse(item.value)) ?? [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)